src/utils/index.ts

```diff
@@ -70,3 +70,29 @@ export const sortByProperty =
       return 0;
     }
   };
+
+export const sortByNestedProperty =
+  <T>(path: string, reverse = false) =>
```

Comment:
Could it be possible to use the `NestedKeyOf` type instead of `string` in path? That would allow IntelliSense suggestions in IDE.

Reply:
Changed to use NestedKeyOf type